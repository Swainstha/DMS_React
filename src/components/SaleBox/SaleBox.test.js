import testSnapFunction from '../../util/testSnapFunction';
import SaleBox from './index'
import React from 'react';
const dummy_data= {
    styleName:'', 
    headerStyle:{fontSize:3}, 
    itemStyle:{border:2}, 
    footerStyle:{color:"yellow"},
    heading:360, 
    title:"title", 
    detail:"detail1", 
    children:"children", 
    subHeadingColor:"black"
}

testSnapFunction("<SaleBox/>", "Snapshot test for PhroductGridItem",<SaleBox {...dummy_data}/> )