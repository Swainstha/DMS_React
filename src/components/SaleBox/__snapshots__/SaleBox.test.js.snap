// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SaleBox/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
